lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java

```diff
@@ -77,6 +79,19 @@ protected DatasetApi(HttpClientWrapper httpClientWrapper, String id, boolean isP
         return getVersionedFromTarget("files", version, List.class, FileMeta.class);
     }
 
+    /**
+     * See [Dataverse API Guide].
+     *
+     * [Dataverse API Guide]:https://guides.dataverse.org/en/latest/api/native-api.html#publish-a-dataset
```

Comment:
```suggestion
     * [Dataverse API Guide]: https://guides.dataverse.org/en/latest/api/native-api.html#publish-a-dataset
```
Markdown will probably not convert it correctly without the space.